src/xdpruntime/xdp-for-windows-runtime.nuspec.in

```diff
@@ -2,15 +2,16 @@
 <!-- Copyright (c) Microsoft Corporation -->
 <package>
     <metadata>
-        <title>XDP for Windows Runtime ({arch})</title>
-        <id>XDP-for-Windows-Runtime.{arch}</id>
+        <id>Microsoft.XDP-for-Windows.Runtime.{arch}</id>
```

Comment:
So is the long term plan to keep using {arch} stuff or replace it with a singleton?

Reply:
Keeping this separate is the most flexible solution.